assembly/globals.ts

```diff
@@ -83,7 +83,7 @@ export function __umulh64(u: u64, v: u64): u64 {
 
 @global
 export function __umulq64(_res: usize, u: u64, v: u64): void {
-  var u1, v1, w0, w1, t;
+  var u1 : u64 , v1 : u64, w0 : u64, w1 : u64, t : u64;
```

Comment:
Remove plz spaces between itentifier and `:` like `u1: u64` for convention with other formatting style. Later I'll add tslint with according rules.